examples/dynamic/views/Html.jsx

```diff
@@ -0,0 +1,33 @@
+var React = require('react');
+var Content = React.createFactory(require('./Content'));
+
+module.exports = React.createClass({
+
+  render: function() {
+    var data = this.props.data;
+
+    // render the content as a dynamic react component
+    var contentHtml = React.renderToString(Content(data));
+
+    // re-render the content as json,
+    // for client-side app initialization
+    var initScript = 'main(' + JSON.stringify(data).replace(/script/g, 'scr"+"ipt') + ')';
```

Comment:
It might be worth a comment here indicating that this is where the security whole is and it might be smart to use a more complete sanitization library.